Headlines/src/Categories/UIImageEffects.swift

```diff
@@ -128,10 +128,11 @@ public extension UIImage {
         return applyBlurWithRadius(10, tintColor: effectColor, saturationDeltaFactor: -1.0, maskImage: nil)
     }
 
-    public func applyBlurWithRadius(_ blurRadius: CGFloat,
-                                    tintColor: UIColor?,
-                                    saturationDeltaFactor: CGFloat,
-                                    maskImage: UIImage? = nil) -> UIImage? {
+    func applyBlurWithRadius(_ blurRadius: CGFloat,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/33ac1bfc9d211087766993200c2602922e761646/Headlines/src/Categories/UIImageEffects.swift#L131"></span>Function should have complexity 10 or less: currently complexity equals 17<br />
<code>cyclomatic_complexity</code> <code>UIImageEffects.swift:131</code></td>
    </tr>
  </tbody>
</table>